src/actions/widgets/index.js

```diff
@@ -1,5 +1,5 @@
 export const CLOCK_TICK = 'clockTick'
 
-const clockTick = time => ({ type: CLOCK_TICK, time })
+const clockTick = time => ( { type: CLOCK_TICK, time } )
```

Comment:
Ew. Let's add an exception to the [space-in-parens](http://eslint.org/docs/rules/space-in-parens) rule for curly braces.
```
{ "exceptions": ["{}"] }
```